### DISCUSSION TITLE
Configure modules from `appsettings.json`

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#14414] [Q&A] [answered]
In version 1.1 I was able to enable and disable modules per tenant using `OrchardCore.<tenant name>.Features`
``` JSON
{
  "OrchardCore": {
    "Default": {
      "State": "Running",
      "RequestUrlHost": null,
      "RequestUrlPrefix": null,
      "Features": [
        "OrchardCore.Mvc.HelloWorld"
      ]
    }
  }
}
```
Does it still work?
How to do such setup?

**Comment by jtkech** [comment]
> Does it still work?

Normally yes.

> How to do such setup?

Normally just update the appsettings.json and then run the app

**Reply by xperiandri** [reply]
I did that but modules run always

**Reply by xperiandri** [reply]
Regardless of the section

**Reply by xperiandri** [reply]
Should I enable some service?

**Reply by jtkech** [reply]
Yes maybe, from memory there is a descriptor manager for all features and one for only configured features, I will look at it.

**Reply by jtkech** [reply, marked as answer]
Okay, this service is registered if you use `.WithTenants()`.
